blog/2024-11-2-app-platform-v12.md

```diff
@@ -0,0 +1,58 @@
+---
+slug: 2024/11/app-platform-v12
+title: App Platform v12
+authors: core
+tags: [app platform, developer tools, webapp, announcement]
+---
+
+# Platform v12 Migration Guide: Vite & React 18
```

Comment:
Putting `release` first as this is a release announcement, but also has migration included, so putting that secondary. 

```suggestion
# Platform v12 release, Vite and React 18 Migration Guide
```